content/post/059-ipfs-camp-sci-fi-fair-videos.md

```diff
@@ -0,0 +1,69 @@
+
+![@achingbrain's npm-in-a-box](/059-ipfs-camp-sci-fi-fair-videos/npm-in-box.jpg)
+_@achingbrain's npm-in-a-box project_
+
+**BONUS:** If you like big IPFS nodes running on small computers (and/or "The IT Crowd"), you may also enjoy Alex's "npm in a box", which packs the entire NPM package repository on a very portable (and sleek looking, if we say so) RPi based IPFS node with a BIG archival grade hard drive. Take this with you on your field trip and your team will never see an npm timeout ever again.
+
+### MetaMask's Network Visualizer
+
+{{< youtube 7HDVgjQnNdM >}}
+_@kumavis shows off the live network visualizer_
+
+The MetaMask crew brought a tool that proved invaluable in their network connectivity diagnostics: a DHT visualizer. Following some wireless network trouble (a real-world scenario to be sure), we were treated to some beautiful visualizations of network connectivity.
+
+### And More
+These are just some of the projects that made it to the Fair: head over to the [offical Fair playlist](https://www.youtube.com/playlist?list=PLuhRWgmPaHtTWCyJcTgLFhsg8Bi2dFPCo) to catch the rest of the videos, as well as the full cuts of the interviews above. Don't miss Berty's Bluetooth Low Energy p2p transport, Brave's one-click in-browser IPFS, and more!
+
+### Lest We Forget
+Our tireless camera crew could only be in one place at a time (we're working on it), which has left some of our favorite projects sadly underdocumented in the video department. Here are some stills that fill in the gaps:
+
+![@MarneeDear's FARS + @gorhgorh's minitel IPFS](/059-ipfs-camp-sci-fi-fair-videos/marnee-minitel.jpg)
+_Marnee Dearman's IPFS-over-Ham Radio FARS project and Jérôme's IPFS Chat on a French Minitel terminal_
+
+![@vasa-develop's 3D collaboration](/059-ipfs-camp-sci-fi-fair-videos/vasa.jpg)
+_Vasa's CRDT-based collaborative 3D modeling interface_
+
+![@gorhgorh's catbot](/059-ipfs-camp-sci-fi-fair-videos/catbot.jpg)
+_Jérôme Loï's libp2p-powered Lazer Catbot (unfortunately no cats were on premises)_
+
+![the elusive @rklaehn](/059-ipfs-camp-sci-fi-fair-videos/actyx.jpg)
+_Actyx's Rüdiger has somehow managed to avoid ALL photos at the fair, so here's a shot of him demoing their amazing industrial automation system from the lightning talks_ 
+
+![@aschmahmann's Fast IPNS](/059-ipfs-camp-sci-fi-fair-videos/ipns.jpg)
+_Last but not least, Adin's "Fast IPNS" demo (it was VERY FAST!)_
+
+## Community Interviews
```

Comment:
@doctorrobinson - can you add a few paragraphs highlighting the community interviews (maybe calling out Netflix and Audius like Arkadiy's examples above)?

Reply:
Dropped it below - @parkan can you add to your update please?